### DISCUSSION TITLE
SNZB-05P

### DISCUSSION THREAD

**Discussion by cpuks** [Koenkk/zigbee2mqtt#23425] [Devices]
Does anyone have this new Sonoff water leak sensor working? It's detected by z2m, interview is fine then no data is pulled off device - no status update for over 24 hrs, tried repairing, removed battery, I've only managed once to get it into leak state.
Seems like recently deconz added support for that device https://github.com/dresden-elektronik/deconz-rest-plugin/pull/7843 but it was a bit wonky.
Pooling also gives an error so once device is connected to network it does nothing:
`z2m: Request 'zigbee2mqtt/bridge/request/device/configure_reporting' failed with error: 'Bind 0x0ceff6fffe69b6ce/1 genPollCtrl from '0x00124b00237ba437/1' failed (AREQ - ZDO - bindRsp after 10000ms)'`

`z2m: Request 'zigbee2mqtt/bridge/request/device/interview' failed with error: 'interview of 'kotłownia czujnik zalania' (0x0ceff6fffe69b6ce) failed: Error: Interview failed because can not get active endpoints ('0x0ceff6fffe69b6ce')'`

`z2m: Request 'zigbee2mqtt/bridge/request/device/configure_reporting' failed with error: 'Bind 0x0ceff6fffe69b6ce/1 genPollCtrl from '0x00124b00237ba437/1' failed (AREQ - ZDO - bindRsp after 10000ms)'`

**Comment by ralfsteck** [comment]
Hi, it works here, but the battery status on the "about" page shows "?", while on the page Details the battery status is "100%". Weird.
